domains/quantum.json

```
@@ -1,9 +1,11 @@
 {
-  "owner": {
-    "username": "RabbitFored",
-    "email": "quantum@theostrich.eu.org"
-  },
-  "record": {
-    "CNAME": "quantum.theostrich.eu.org"
-  }
+    "owner": {
+        "username": "lakshinhemachandrna",
+        "email": "quantumdev-official@outlook.com"
+    },
+    "record": {
+        "MX": ["mx1.improvmx.com", "mx2.improvmx.com"],
+        "TXT": ["v=spf1 include:spf.improvmx.com ~all"]
+        "CNAME": ["lakshinhemachandran.github.io"]
```

CNAME record shouldnt be in an array
```suggestion
        "CNAME": "lakshinhemachandran.github.io"
```